README.md

```diff
@@ -22,6 +22,7 @@ The packages in this library implement the various components of the CONIKS syst
 - ``merkletree``: Merkle prefix tree and related data structures
 - ``keyserver``: Server-side protocol implementation
```

Comment:
Now that we've split up the code, I think the description here should be something like "Key server executable" or "Key server reference implementation".